_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Version 1
+---
+
+We are happy to release version 1 of MDAnalysis!
+
+
+MDAnalysis 1.x is a stable legacy platform that supports 
+
+-  Python 2.7 (on Linux and MacOS only), 
+-  Python 3.5, 3.6, 3.7 and 3.8 (on Windows (32- and 64-bit), MacOS, and Linux).
+
+The API will not change for any upcoming 1.x versions.
+This release completes the first phase of our [roadmap][].
+From here on, we will focus on developing version 2 of MDAnalysis, which will support Python 3.6+ only, and will include major API changes and improvements.
+
+
+The 1.0.0, 1.0.1 and 1.1.1 versions of MDAnalysis are the product of more than 18 months effort.
+They contain a multitude of **fixes**, [**deprecations**](#deprecations), and **new features**.
+We highlight important changes below, but users are strongly encouraged to read the [CHANGELOG][] for full details.
+All users are recommended to upgrade to 1.1.1, as it includes important fixes to neighbor searching, a fundamental component of many analyses in MDAnalysis.
+
+In [Dependencies](#dependencies) we list the required dependencies of MDAnaysis version 1.
+
+
+## Notable new additions
+
+We now support several new formats:
+- [FHI-aims](https://userguide.mdanalysis.org/stable/formats/reference/in)
+- [NAMD binary coordinates](https://userguide.mdanalysis.org/stable/formats/reference/coor)
+- [ITP files from GROMACS](https://userguide.mdanalysis.org/stable/formats/reference/itp)
+
+We have also added [*converters*](https://docs.mdanalysis.org/stable/documentation_pages/converters) to and from other popular analysis packages.
```

Comment:
```suggestion
We have also added [*converters*](https://docs.mdanalysis.org/1.1.1/documentation_pages/converters) to and from other popular analysis packages.
```

Where possible I think we should be linking to 1.1.1 not stable. That way when 2.0 gets released these links don't suddenly start pointing at the wrong place?

@lilyminium I'll default to whatever you think is best here, also is there a way to do something similar for the userguide?

Reply:
merging these after discussion with @lilyminium on discord